tests/test_technically_eligible_land.py

```diff
@@ -1,21 +1,19 @@
 import pytest
 import numpy as np
 
-from scripts.technical_eligibility import Eligibility, _determine_eligibility, GlobCover
-
+from scripts.technical_eligibility import _determine_eligibility
```

Comment:
It seems this function should move into the `lib`, so that this import becomes unnecessary.

Reply:
I thought about it, but unlike the other function in the `lib`,  '_determine_elegibility' relies heavily on snakemake inputs. Therefore it seems a bit odd to have it in there. This test does function in the current setup.

Reply:
Are you sure? The function `_determine_elegibility` (the one with the underscore!) does not rely on Snakemake at all, does it?

I'm a little picky here because this is one of the central functions of this workflow.